lib/components/MapFileTile.js

```diff
@@ -0,0 +1,79 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {
+  ViewPropTypes,
+  View,
+} from 'react-native';
+
+import decorateMapComponent, {
+  USES_DEFAULT_IMPLEMENTATION,
+  SUPPORTED,
+} from './decorateMapComponent';
+
+// if ViewPropTypes is not defined fall back to View.propType (to support RN < 0.44)
+const viewPropTypes = ViewPropTypes || View.propTypes;
+
+const propTypes = {
+  ...viewPropTypes,
+
+  /**
+   * The order in which this tile overlay is drawn with respect to other overlays. An overlay
+   * with a larger z-index is drawn over overlays with smaller z-indices. The order of overlays
+   * with the same z-index is arbitrary. The default zIndex is -1.
+   *
+   * @platform android
+   */
+  zIndex: PropTypes.number,
+  /**
+   * The maximum zoom level for this tile overlay.
+   *
+   */
+  maximumZ: PropTypes.number,
+
+  /**
+   * The minimum zoom level for this tile overlay.
+   *
+   */
+  minimumZ: PropTypes.number,
+
+  /**
+   * Corresponds to MKTileOverlay canReplaceMapContent.
+   *
+   * @platform ios
+   */
+  shouldReplaceMapContent: PropTypes.bool,
+
+  /**
+   * (Optional) Tile size for iOS only, default size is 256 * 256.
+   *
+   * @platform ios
+   */
+  tileSize: PropTypes.number,
+};
+
+class MapFileTile extends React.Component {
+  render() {
+    const AIRMapUrlTile = this.getAirComponent();
+    return (
+      <AIRMapUrlTile
+        {...this.props}
+      />
+    );
+  }
+}
+
+MapFileTile.propTypes = propTypes;
+
+export default decorateMapComponent(MapFileTile, {
+  componentType: 'FileTile',
+  providers: {
+    google: {
+      ios: SUPPORTED,
```

Comment:
This lines (72-73) means that offline tiles are compatible with google maps on android and ios. Unless I am missing something, it seems to not be the case.

Reply:
You're right about this. I just based it on MapUrlTile.js, without going into details. Google map provider shouldn't be supported at all. I don't know react and js part in this lib well - should I remove entire `google` block?

Reply:
Yes, please. Thanks.

Reply:
Done.